browser/ai_chat/BUILD.gn

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
it seems like all of this should go in components/ai_chat under components/ai_chat/core for everything except ai_chat_tab_helper which should go under components/ai_chat/content see https://www.chromium.org/developers/design-documents/layered-components-design/#layered-components

Reply:
moved now, except I didn't put `ai_chat_tab_helper` under `content/` yet, it's moved to `components/ai_chat` though